rs/nns/governance/canister/governance.did

```diff
@@ -642,9 +683,14 @@ type NeuronInfo = record {
   retrieved_at_timestamp_seconds : nat64;
   visibility : opt int32;
   known_neuron_data : opt KnownNeuronData;
+
+  // Deprecated. Use either deciding_voting_power or potential_voting_power instead.
```

Comment:
> Will it be removed

I'd rather not, because this field is not opt. In any case, I don't think we need to decide that soon.

> will it contain the same value as the deciding_voting_power field?

Not sure. Need to figure this out. Another options is for this to instead have the same value as potential_voting_power. potential_voting_power seems more natural to me, because in the past, refresh time was never used anywhere, but I think both options are valid, and if deciding_voting_power is better for clients in some way, then, I would go with that.

Another orthogonal change we can do here is zero this out when dissolve delay is < 6 months. If we really wanted to minimize disruption though, this would continue to NOT be zeroed out. This is not so convenient from an implementation point of view though. The most convenient thing is to say either that this = deciding_voting_power or that this = potential_voting_power.

Reply:
I think this is an important one. I actually assumed that we go the way you describe in the last paragraph and leave the deprecated field as is (non-zero even if a neuron's dd is <6m). 
This is what I told @dylancm4 . So if we decide something else, let's quickly check with him?

Reply:
Did he say he prefers no zero-ing out? Or does he not care? I think he shouldn't care, but I don't want to put words in his mouth.

Reply:
Does Dylan care what the behavior is? Or does he simply want to know what it will be?

Reply:
Maksims and I discussed over zoom. We think this should have decision_voting_power.

We did not discuss zero-ing out though.

The most convenient thing for the backend impl is to indeed have zero-ing out. This could be viewed as a behavior change that clients **might** not like. I suggest that we go with zero-ing out, and if not body complains, go for zero-ing out.

> It's better to ask for forgiveness than permission.
> --Various

Reply:
The dashboard team might not have time to make all changes right away, so it would be good to make sure we don't break them until they have time to do it.
But even if they had time, with the recent learnings I think it is always good to have a way forward so that changes can be released one at a time, right? So would be good to make sure that the releases don't break them and we have an order of releases that gets us to the right thing.

Reply:
I will keep trying to get confirmation from Dylan about this (i.e. whether they are ok with zeroing out voting_power). FWIW, I think it is very likely that this will not break them. For now, Lara and I resolved over zoom that we'll go with this optimistic assumption, and course correct if we later find out zero-ing out does not work for everybody.

Reply:
Zeroing out the deprecated metric when dissolve delay is < 6 months will not affect the ICP Dashboard. Of course, we will migrate to using the new fields instead of the deprecated one, but even if we didn't do this right away, the proposed implementation would work fine in the interim.